documentation/projects/proposals/rekognition_data/20240530-implementation_plan_augment_catalog_with_rekognition_tags.md

```diff
@@ -118,24 +126,26 @@ have a demographic context in the following categories:
 - Sexual orientation
 - Nationality
 - Race
+- Marital status
 
 There are other categories which might be useful for search relevancy and are
 less likely to be applied in an insensitive manner. These labels **should not**
-be excluded. Some examples include:
+be excluded, unless they are otherwise gendered (e.g. "stewardess", "actress",
+etc.). Some examples include:
 
 - Occupation
-- Marital status
 - Health and disability status
 - Political affiliation or preference
 - Religious affiliation or preference
```

Comment:
In [this comment thread](https://github.com/WordPress/openverse/pull/4417#discussion_r1623673094) there was discussion about filtering all demographic information related to human subjects, which would include most of these I think. What do you think?

Reply:
I thought the comment thread landed on filtering only genered terms from those, but keeping others? Maybe we don't need to have such a strict policy here and should overall suggest considering these when reviewing the list of tags? This feels like a _very_ abstract way of talking about an actually concrete set of tags. There aren't even tags related to health or disability status in Rekognition, as far as I saw, for example. Could we go case-by-case during the review of the tags, trusting that we all agree on the principle of what we're getting at here (excluding tags that presuppose something about the subject that cannot be reliably derived from an aesthetic judgement)?

Reply:
Yea I added a provision specifically talking about the gendered aspect of some of the demographic pieces in this list, but I do think having these would be useful if they're provided. We'll certainly get a better sense of what might be better to _add_ to the list of things we're filtering once we actually go through the various provider lists, as Sara mentions.

Reply:
I don't see where we landed on that -- Sara and I both singled out the religious affiliation in particular as being problematic for that reason (presuppoing something that can't be determined just from looks), and Zack's and my last comments are both in support of filtering all demographic information. I agree that these are super abstract categories and that it should be considered case-by-case in the review of the tags, but won't the reviewers be using this IP to make their judgments? Explicitly singling these categories out as okay/not worth filtering in the IP _does_ feel like a strict policy as opposed to leaving it to case-by-case review, and if I was reviewing the list of tags I would assume this meant we've agreed to definitely include religious affiliations, for instance 🤷‍♀️ 

If it's just the case that Rekognition doesn't have any tags related to these categories anyway, so it's not a problem _this time_, I would still want our documentation to accurately reflect the sorts of tags we want to filter when we eventually encounter them.

This still wasn't a blocker for me, particularly because it's easy to change down the road, but this response is more confusing to me. I'm not sure I would know which tags to filter if I were reviewing the list.

Reply:
Thanks for bringing this back up Staci, and I'm sorry that I missed this! I'll come back with an update to the IP to clarify this, specifically around having it be more case-by-case but suggesting to avoid all demographics entirely.